content/en/docs/spinnaker/using-dinghy.md

```diff
@@ -451,6 +451,165 @@ Note how the `requisiteStageRefIds` is overwritten while calling the module so t
+        {{ local_module "/local_modules/stage.minimal.wait.localmodule" }},
+        {{ local_module "/local_modules/stage.minimal.wait.localmodule" "waitname" "localmodule overwrite-name" "waitTime" "100" }},
+        {{ module "stage.minimal.wait.module" "waitname" "global module overwrite-name" "waitTime" "100" }}
+      ]
+    }
+  ]
+}
+```
+
+`stage.minimal.wait.localmodule` and `stage.minimal.wait.module` have the same content:
+``` json
+{
+  "name": "{{ var "waitname" ?: "Local Module Wait" }}",
+  "waitTime":  "{{ var "waitTime" ?: 10 }}",
+  "type": "wait"
+}
+```
+
+The rendered file looks like this:
+
+``` json
+{
+  "application": "localmodules",
+  "globals": {
+    "waitTime": "42",
+    "waitname": "localmodule default-name"
+  },
+  "pipelines": [
+    {
+      "application": "localmodules",
+      "name": "Made By Armory Pipeline Templates",
+      "stages": [
+        {
+          "name": "localmodule default-name",
+          "waitTime": "42",
+          "type": "wait"
+        },
+        {
+          "name": "localmodule overwrite-name",
+          "waitTime": "100",
+          "type": "wait"
+        },
+        {
+          "name": "global module overwrite-name",
+          "waitTime": "100",
+          "type": "wait"
+        }
+      ]
+    }
+  ]
+}
+```
+
+Note that `local_module` and `module` can be combined in one dinghyfile.
+
+
+### Using local modules and modules
+
+As shown in the previous example, you can use `local_module` and `module` without any issues from any `dinghyfile`. The rendering functionality is implemented in the Armory CLI tool, so you can validate your dinghyfiles with `local_module`.
+
+
+### Local modules limitations
+
+You can reference `module` and `local_module` from any `local_module`. However, you cannot call a `local_module` from a `module`. Given this scenario:
+
+
+`my_repository`:
+```
+.
+├── dinghyfile
+└── local_modules
+    └── stage.minimal.wait.localmodule
+
+```
+
+`template_repository`:
+
+```
+.
+└── stage.minimal.wait.module
+```
+
+And inside the `dinghyfile`, `stage.minimal.wait.localmodule` and `stage.minimal.wait.module` you can see this.
+
+`dinghyfile`
+
+``` json
+{
+  "application": "localmodules",
+  "globals": {
+      "waitTime": "42",
+      "waitname": "localmodule default-name"
+  },
+  "pipelines": [
+    {
+      "application": "localmodules",
+      "name": "Made By Armory Pipeline Templates",
+      "stages": [
+        {{ module "stage.minimal.wait.module" }}
+      ]
+    }
+  ]
+}
+```
+
+`stage.minimal.wait.module`:
+``` json
+{
+  "name": "{{ var "waitname" ?: "Local Module Wait" }}",
+  "waitTime":  "{{ var "waitTime" ?: 10 }}",
+  "type": "wait"
+},
+{{ local_module "/local_modules/stage.minimal.wait.localmodule" }}
+```
+
+When Dinghy sees that there's a `local_module` being sent inside a `module`, it throws the following error message: `calling local_module from a module is not allowed`. For the given example scenario, the following error occurs:
```

Comment:
Can we display the file names & paths being referenced to help in debugging? Or is this what's happening in the next section?